src/main/java/xbot/common/controls/sensors/wpi_adapters/LaserCANWpiAdapter.java

```diff
@@ -46,10 +52,18 @@ public LaserCANWpiAdapter(
     @Override
     public void updateInputs(LaserCANInputs inputs) {
         LaserCanInterface.Measurement measurement = laserCan.getMeasurement();
-        if (measurement != null) {
+        if (measurement != null && measurement.status == LaserCan.LASERCAN_STATUS_VALID_MEASUREMENT) {
```

Comment:
This change is what tripped up the previous PR. The jumps to 0 were being handled as valid data. Instead, report an Optional.empty() when the data is invalid, to disambiguate between 0 and invalid data.